kingpin/actors/aws/elb.py

```diff
@@ -41,6 +41,23 @@
 EXECUTOR = concurrent.futures.ThreadPoolExecutor(10)
 
 
+def dry(dry_message):
+    log.info('Creating _skip_on_dry decorator with "%s"' % dry_message)
+    def _skip_on_dry(function):
+        log.info('Decorating function "%s" with _skip_on_dry' % function)
+        @gen.coroutine
+        def wrapper(self, *args, **kwargs):
+            if self._dry:
+                self.log.info(dry_message)
```

Comment:
you probably need to do some magic with `inspect.getargspec` here to make this work.